newIDE/app/src/MainFrame/EditorContainers/HomePage/LearnSection/MainPage.js

```diff
@@ -198,6 +204,14 @@ const MainPage = ({
           </GridList>
         </Line>
       </SectionRow>
+      {shouldShowInAppTutorialButtons && (
+        <SectionRow>
+          <Text noMargin size="section-title">
+            <Trans>In-app tutorials</Trans>
```

Comment:
Is "In-app" something that users will understand? The vocabulary used is generally like "Guided tours" or something else.

Reply:
yeah good question, let's go for "guided lesson"